examples/templates/angular/src/app/app.component.html

```diff
@@ -0,0 +1 @@
+<app-hot-table></app-hot-table>
```

Comment:
Maybe we could flatten the example if we move Handsontable from a separate component to the `app.component` from our template? @swistach WDYT, will it be usefull to have additional component for future examples?

Reply:
Done https://github.com/handsontable/handsontable/pull/7588/commits/a8e22a6d719a54ea5601d23329f2da74a902cc9e